power-pay-frontend/src/hooks/useStorage.ts

```diff
@@ -0,0 +1,27 @@
+//useState and useEffect hooks from React
+import { useState } from 'react';
+
+//UseStorageProps<T>: An interface defining the props for the useStorage hook
+interface UseStorageProps<T> {
+  initialValue?: T  | undefined;
+}
```

Comment:
You removed the information that was really interesting, the key. We use it to save our information to the storage and to read them too.